packages/plugin-credentials/src/credentials/browser.js

```diff
@@ -27,6 +27,11 @@ const Credentials = SparkPlugin.extend(Object.assign({}, common, {
     isLoggingIn: {
       default: false,
       type: `boolean`
+    },
+    // variable to check if there are user credentials with which auto login is possible, hence the client needs to wait until the credentails validation is completed.
+    isValidatingUser: {
```

Comment:
How about flipping the states of this boolean and naming it something more like `queryStringProcessed`?